cmd_set.go

```diff
@@ -672,17 +678,34 @@ func (m *Miniredis) cmdSscan(c *server.Peer, cmd string, args []string) {
 			c.WriteError(ErrWrongType.Error())
 			return
 		}
-
 		members := db.setMembers(opts.key)
 		if opts.withMatch {
 			members, _ = matchKeys(members, opts.match)
 		}
-
-		c.WriteLen(2)
-		c.WriteBulk("0") // no next cursor
-		c.WriteLen(len(members))
-		for _, k := range members {
-			c.WriteBulk(k)
+		low := opts.cursor
+		end := low + opts.count
+		for {
```

Comment:
Why is the loop here?  This appears like it will write all members of the set out

Reply:
I was testing the testing functions and I forgot to remove it after I was done and properly update the tests. It should be correct now.